lib/helpers/json-to-html-data/pretty-status.js

```diff
@@ -0,0 +1,9 @@
+module.exports = function (status) {
+  var mapping = {
+    'permanent collection': 'Permanent collection',
+    'loan: wellcome trust': 'Loan: Wellcome Trust',
+    'loans: historic and loan': 'Loan'
```

Comment:
yes I should have look a bit better at the issue, it's true that "historic and loan" as a value doesn't make any sense, thanks @Danwhy